CHANGELOG.md

```diff
@@ -17,7 +17,7 @@
 
 ## API changes
 - `SurrogateGenerator`s now have a field `s` into which surrogates are generated, avoiding
-    unneccessary memory allocations. The
+    unnecessary memory allocations. The

```

Comment:
Good catch, but the correction is also not correct. This should be ["unnecessary"](https://www.oed.com/dictionary/unnecessary_adj?tab=factsheet#16604305) (with two n-s and two s-s).

Reply:
Ops, I'm sorry. You're perfectly right.

Reply:
> Hi @kahaaga, I'm not sure I understand. Isn't that the correction I've pushed?

You are absolutely correct. It appears I'm the jet-lagged one. Forget my comment entirely!